functions/api/routes/community/communityCategoryPostsGET.js

```diff
@@ -1,9 +1,79 @@
+const dayjs = require('dayjs');
+const customParseFormat = require('dayjs/plugin/customParseFormat');
+const util = require('../../../lib/util');
+const statusCode = require('../../../constants/statusCode');
+const responseMessage = require('../../../constants/responseMessage');
+const asyncWrapper = require('../../../lib/asyncWrapper');
+const db = require('../../../db/db');
+const { communityDB } = require('../../../db');
+const dummyImages = require('../../../constants/dummyImages');
 /**
  *  @route GET /community/categories/:communityCategoryId
  *  @desc 커뮤니티 카테고리별 게시글 조회
  *  @access Private
  */
 
-module.exports = async (req, res) => {
+module.exports = asyncWrapper(async (req, res) => {
+  const { userId } = req.user;
   const { page, limit } = req.query;
-};
+  const { communityCategoryId } = req.params;
+
+  const dbConnection = await db.connect(req);
+  req.dbConnection = dbConnection;
+
+  dayjs().format();
+  dayjs.extend(customParseFormat);
+
+  // category id가 존재하지 않는 경우
+  const isExistingCategory = await communityDB.isExistingCategory(
+    dbConnection,
+    communityCategoryId,
+  );
+  if (!isExistingCategory) {
+    return res
+      .status(statusCode.NOT_FOUND)
+      .send(util.fail(statusCode.NOT_FOUND, responseMessage.NO_CATEGORY));
+  }
+
+  const totalItemCount = await communityDB.getCommunityCategoryPostsCount(
+    dbConnection,
+    userId,
+    communityCategoryId,
+  ); // 총 게시글 수
+  const totalPageCount = Math.ceil(totalItemCount / limit); // 총 페이지 수
+  const currentPage = +page; // 현재 페이지
+  const isLastPage = totalPageCount === currentPage; // 마지막 페이지인지 여부
```

Comment:
이 변수는 반환할 때말고 안쓰이는 것 같은데 아래처럼 하는게 어떨까요?
```suggestion
   res.status(statusCode.OK).send(
    util.success(statusCode.OK, responseMessage.READ_COMMUNITY_CATEGORY_POSTS_SUCCESS, {
      posts: result,
      currentPage,
      totalPageCount,
      totalItemCount,
      isLastPage: totalPageCount === currentPage,
    }),
  );
```

Reply:
맞아요 반환할 때만 쓰입니다! 반영하겠습니다 👍